<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormFunc</class>
 <widget class="QWidget" name="FormFunc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Przeglądarka muzyki</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="3" column="10" colspan="2">
    <widget class="QPushButton" name="playB">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Odtwarzaj w
QMPlay</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="13">
    <widget class="QLineEdit" name="titleE"/>
   </item>
   <item row="3" column="4" colspan="3">
    <widget class="QLabel" name="pagesL"/>
   </item>
   <item row="3" column="9">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="13">
    <widget class="QPushButton" name="downloadB">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Pobierz na
dysk</string>
     </property>
    </widget>
   </item>
   <item row="2" column="10" colspan="2">
    <widget class="QPushButton" name="dwnPB">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Ścieżka
pobierania</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="9">
    <widget class="QLabel" name="infoL">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="13">
    <widget class="QPushButton" name="clrB">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Wyczyść
wyniki</string>
     </property>
    </widget>
   </item>
   <item row="0" column="13">
    <widget class="QPushButton" name="searchB">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Szukaj</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="prevB">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QPushButton" name="nextB">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="14">
    <widget class="QTableWidget" name="titlesW">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Tytuł</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Długość</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rozmiar</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Użytkownik</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="currPageB">
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="14">
    <widget class="QWidget" name="dwnW" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="5" column="0" colspan="2">
       <widget class="QProgressBar" name="progressB">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="abortB">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="dwnTitleL">
        <property name="text">
         <string>Pobieranie</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="dwnInfoL">
        <property name="text">
         <string>Szybkość:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QListWidget" name="queueW">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>75</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>titleE</tabstop>
  <tabstop>searchB</tabstop>
  <tabstop>titlesW</tabstop>
  <tabstop>prevB</tabstop>
  <tabstop>currPageB</tabstop>
  <tabstop>nextB</tabstop>
  <tabstop>dwnPB</tabstop>
  <tabstop>playB</tabstop>
  <tabstop>clrB</tabstop>
  <tabstop>downloadB</tabstop>
  <tabstop>queueW</tabstop>
  <tabstop>abortB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
